### ISSUE TITLE
Is this going to be added to winlator mali new versions?

### ISSUE THREAD

**Issue by CODERMJ07** [Fcharan/Wine-Hangover-Patched#8]
Is it possible ?

**Comment by khanh-it** [comment]
I don't think it is possible as of right now because Winlator has started emulators using Proot. The Mesa Native Wrapper is a Bionic Libc build (run directly on Termux Native); it doesn't support chroot/proot.

**Comment by CODERMJ07** [comment]
ok

**Comment by khanh-it** [comment]
You can try MiceWine emulator. It has the same design as installer script (not working for Mali yet)
https://github.com/KreitinnSoftware/MiceWine-Application

**Comment by CODERMJ07** [comment]
I tried and it didn't quite work for me.

**Comment by CODERMJ07** [comment]
winlator did add a new vortek driver but it didn't work for me. But it seems like adding this wrapper should be possible.

**Comment by khanh-it** [comment]
Winlator 9.0 vortek driver didn't work for me as well.
Here is my test result
https://github.com/xMeM/vulkan-wsi-layer/issues/41
